config/environments/migration.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+require Rails.root.join("config/environments/production")
+
+Rails.application.configure do
+  config.log_level = :debug
+  config.ssl_options = { redirect: { exclude: ->(request) { request.path.include?("/healthcheck") } } }
```

Comment:
Should the excluded path be `/check`? It seems that's the case in all the other envs.

Reply:
ah yeah, my bad. It's `/healthcheck` on NPQ! Thanks, will force push in a sec